cdilemna/core.py

```diff
@@ -87,14 +89,34 @@ class JoinGame(BaseModel):
     players[user.id] = user
     # TODO ensure that we only add the player if the game is in 'SETUP' status
     game.add_player(user.id)
-    EVENT_QUEUE.put(f'{player_name} joined the game {game_id}')
+    await EVENT_QUEUE.put(
+        {
+            'type': 'GAME',
+            'message': f'{player_name} joined the game {game_id}',
+            'game_id': f'{game_id}'
+        }
+    )
     return {
         'user_id': f'{user.id}',
         'players': f'{game.get_players()}',
         'game_owner': f'{game.get_owner_id()}',
         'game_id': f'{game_id}'
     }
 
+@app.websocket('/ws/game/{game_id}')
+async def websocket_endpoint(websocket: WebSocket, game_id: int):
+    await websocket.accept()
+    game_queue = game_queues[game_id]
+    game = games[game_id]
+    await websocket.send_json(game.__dict__)
+    while True:
+        await asyncio.sleep(5)
+        message = {'message': 'hello'}
+        await websocket.send_json(message)
+        # event = await game_queue.get()
+        # await websocket.send_text(f'{event}')
+        # game_queue.task_done()
```

Comment:
I really struggled to get this to work. There is probably a better way to accomplish this though.